src/main/resources/templates/Landing.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html>
+  <title>Online HTML Editor</title>
+  <link rel="stylesheet" href="mystyle.css">
+
+  <head>
+    <div class="textBox">

```

Comment:
This div and its contents should be under the body tag

Reply:
Will discuss with Nolan